src/components/SongList/Index.js

```diff
@@ -0,0 +1,18 @@
+import './SongList.css';
+
+export default function SongList({ tracks }) {
+  
+    return (
+        <>
```

Comment:
you can remove this fragment. You are already returning only one element, the ul.